pkg/virt-handler/node-labeller/cpu_plugin_test.go

```diff
@@ -1,3 +1,4 @@
+//go:build amd64
 // +build amd64
```

Comment:
These tags can be removed since with this PR the code will only compile with 1.17.

Reply:
I'm not familiar with them. Are you saying that I can remove both
```
//go:build amd64
// +build amd64
```
?
Can you explain me the reason? Thanks
Another point that I noticed: in the `go.mod` is still defined that the `minimum-go-version` is 1.16. Should I upgrade it?

Reply:
> Can you explain me the reason? Thanks

I only mean the `+build` tags. 1.17 introduces the new format.

> Another point that I noticed: in the go.mod is still defined that the minimum-go-version is 1.16. Should I upgrade it?

Afaik the new k8s dependencies only compile with 1.17, so yes :+1: